README.md

```diff
@@ -39,10 +51,21 @@ def mystery2(n)
 end
 ```
 
-- What is mystery2(123)?
-- What is mystery2(9005)?
-- What is mystery2(-123)?
+- What is mystery2(123)? 6
+- What is mystery2(9005)? 14
+- What is mystery2(-123)? -123
 - _Added Fun: How could we make `mystery2(-123)` work the way we might expect it to work instead of the way it does?_
+  We could change the first part of the if statement to an absolute statement,
+
+  ```
+  def mystery2(n)
+    if n.abs < 10
+      return n
+    else
+      return (n%10) + mystery2(n/10)
+    end
+  end
```

Comment:
Hmm... This will definitely change the behavior of what the method does, but perhaps I should have specified what "the way we might expect it to work" is. I wold expect mystery2(-123) to produce -6 not 6. If you expected 6, then your solution is correct. If you wanted -6, then it's not.